README.md

```diff
@@ -359,297 +359,372 @@ that the Kotlin code has been improved, but the underlying Java library remained
     <td>cloudflare</td>
     <td>4.16.0.0</td>
     <td> 
-
+ 
```

Comment:
(minor suggestion)

Can we add a markdown comment above this table with a warning that this table is auto-generated?